testdata/ref/table.md

```diff
@@ -0,0 +1,9 @@
+# We can use tables too
+
+Glorious days ahead! 
+
+| A | B | C |
+|---|---|---| 
+| 1 | 1 | 1 | 
+| *foo* is a common _term_ | _bar_ | **baz** |
+| [link](https://github.com) | `code` | |
```

Comment:
Should we add the various table column variants as well? e.g. centering and whatnot

Reply:
I think adding the column variants here is just to:

1. Demonstrate the rendering doesn't implode, even if we don't implement it correctly
2. Demonstrate the diff between "we didn't implement it" and "we've implemented" better at review time, when we get around to adding it

WDYT?